src/table/utils/handle-key-press.ts

```diff
@@ -1,15 +1,30 @@
+import { stardust } from '@nebula.js/stardust';
+import React from 'react';
```

Comment:
We normally import react first

Reply:
ah yes, this was added automatically. I'll move it

Reply:
done